ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
"Format" is pretty much implied by the fact that this is a Protocol Buffers file.

Just _wire.proto_ would be fine, IMO.

Reply:
Or `wire_kvutils.proto` if you want to keep the context in the name.

Reply:
OK, I renamed it to `submission.proto` and put it under the package `kvutils.wire`, turning off generating an outer class by protoc.